types/v56/types.go

```diff
@@ -2597,6 +2597,7 @@ type QueryResultVMRecordType struct {
 	HostName                 string    `xml:"hostName,attr,omitempty"` // HostName=Hypervisor of virtual machine
 	Link                     []*Link   `xml:"Link,omitempty"`
 	MetaData                 *Metadata `xml:"Metadata,omitempty"`
+	VmNameInVc               string    `xml:"vmNameInVc,omitempty"`
```

Comment:
This is an XML attribute

```suggestion
	VmNameInVc               string    `xml:"vmNameInVc,attr,omitempty"`
```